src/main/java/Duke.java

```diff
@@ -1,10 +1,312 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.Files;
+
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
 public class Duke {
+    private final String VERSION_NUMBER = "1.0.0";
+    private final Scanner SC = new Scanner(System.in);
+    private final Path DUKE_DATA_FILE_PATH = Paths.get("data", "duke.txt");
+    private final Path DUKE_DATA_DIR_PATH = Paths.get("data");
+    private final String NEW_LINE = "\n";
+    private final String HORIZONTAL_LINE =
+            "    ____________________________________________________________";
+    private final String PADDING = "      ";
```

Comment:
Should non-constant fields be written in camelCase instead?

Reply:
They are constant fields